docs/actions.md

```diff
@@ -607,7 +607,7 @@ You can create a OpenWhisk action called `helloSwift` from this function as
 follows:
 
 ```
-wsk action create helloSwift hello.swift
+wsk action create helloSwift hello.swift --kind swift:3.1.1
```

Comment:
Hum I'm not a fan that we have :default and would prefer to be clear and explicit of the version being.
I think I agree to leave it out to have less noise for the folks getting started, and we can have separate discussion across all languages and the use of default in the docs.